paper.md

```diff
@@ -328,11 +328,56 @@ From Table I & Figure III, it can be seen how the portability of Popper workflow
-
-### **Generic workflow execution engines**
-
-### **Container native workflow execution engines**
+The problem of implementing multi-container workflows as described in @Sec:intro is addressed by several existing tools.
+We briefly survey some of these tools and technologies and compare them with Popper 2.0 by grouping them in categories.
+
+## Workflow definition languages
+
+Standard workflow definition laguages like CWL, OpenWDL and YAWL provide an engine agnostic interface for specifying workflows declaratively. 
+Being engine agnostic, different workflow engines can adopt these languages as these workflow languages provide a plethora of useful syntactic elements to support a wide range workflow engine features.
+Some of these workflow definition languages provide syntax for fine-grained control of resources by the users like defining the amount of CPU and memory to be allocated to each step, specifying scheduling policies, etc.
+Most of these languages support syntax for integration with various computing backends like container engines (docker, uDocker, singularity), HPC clusters (HTCondor, LSF, SLURM), cloud providers (AWS, GCP, Azure), Kubernetes, etc.
+For an user whose primary goal is to automate running a set of containerized scripts in sequence, learning these new workflow languages and syntaxes might add to the overall complexity.
+One of Popper's primary goals is to minimize the workflow language overhead as much as possible by allowing users to specify workflows using vanilla YAML syntax, thus keeping the learning curve flat and preventing sources of confusion.
+
+## Workflow execution engines
+
+Workflow execution engines can be categorized in several different categories.
+In this section, we have discussed three frequently used catergories of workflow execution engines namely Generic, Cloud Native and Container Native and compared their pros and cons with Popper.
+
+### Generic workflow execution engines
+
+Let us take a relatively simple scenario where users have a list of single purpose tasks as scripts and they want to automate running them in containers in some sequence.
+In order to accomplish this goal of running a list of containerized tasks using existing workflow engines, users need to learn a specific workflow language, deploy a workflow engine service and learn executing workflows on this service.
+These tasks may not be always trivial to accomplish, if we assume the only thing users should care about is writing experimentation scripts and running them inside containers.
+Few examples of this category are stable and mature scientific workflow engines like Nextflow, Pegasus and Taverna which have recently introduced native support running steps in software containers;
+Industry standard workflow engines like Apache Airflow and Luigi [@_luigi_] which require specifying workflows using programming languages and also provide pluggable interfaces that require the installation of separate plugins;
```

Comment:
Given that there's no definition of "Industry standard", it could be removed and instead say "Popular workflow engines..."